client/state/signup/verticals/reducer.js

```diff
@@ -4,14 +4,22 @@
  * Internal dependencies
  */
 import { createReducer } from 'state/utils';
-import { SIGNUP_VERTICALS_SET } from 'state/action-types';
+import { SIGNUP_VERTICALS_SET, SIGNUP_VERTICALS_REQUEST } from 'state/action-types';
 
 const verticals = createReducer(
 	{},
 	{
+		[ SIGNUP_VERTICALS_REQUEST ]: state => {
+			return {
+				...state,
+				isFetching: true,
```

Comment:
This feels like an unexpected location in the tree for storing a `isFetching` variable. I expected it to be in a separate subtree of the state.

Reply:
Totally agree. Will fix this.